src/ckeditor/ckeditor.component.ts

```diff
@@ -251,11 +253,19 @@ export class CKEditorComponent implements AfterViewInit, OnDestroy, ControlValue
 
 	ngAfterViewInit(): void {
 		getEditorNamespace( this.editorUrl ).then( () => {
+			// Check if component instance was destroyed before `ngAfterViewInit` call (#110).
+			// Here, `this.instance` is still not initialized and so additional flag is needed.
+			if( this._destroyed ) {
+				return;
+			}
+
 			this.ngZone.runOutsideAngular( this.createEditor.bind( this ) );
 		} ).catch( window.console.error );
 	}
 
 	ngOnDestroy(): void {
+		this._destroyed = true;
```

Comment:
I am not quite sure, but I think the destroyed should be only set if there is an instance which exists? In order to that would it make sense to just check, on top, if the instance does not exist anymore?

Reply:
No the point is to avoid creation of instance if ondestroy is call before the end of the promise execution